pkg/cloud/ibmcloud/deployer.go

```diff
@@ -89,3 +92,34 @@ func (d *DeploymentClient) GetKubeConfig(cluster *clusterv1.Cluster, master *clu
 	}
 	return strings.TrimSpace(parts[1]), nil
 }
+
+func getSSHKeyFile(homeDir string) string {
+	defaultKey := homeDir + "/.ssh/id_ibmcloud"
+	envPrivateKey, ok := os.LookupEnv("IBMCLOUD_HOST_SSH_PRIVATE_FILE")
+	if !ok || (0 == strings.Compare(envPrivateKey, "")) {
+		return defaultKey
+	}
+	klog.V(3).Infof("Found environment variable SSH private file %s ", envPrivateKey)
+	exist, _ := FileExists(envPrivateKey)
+	if exist {
+		return envPrivateKey
+	}
+	pwd, err := os.Getwd()
```

Comment:
I just noticed that there is not need for the code after this line.

If it is absolute path or relative path, it is fine to return the file name by `FileExists` checking.

Reply:
Updated code to return the value user defined. Thanks for comments.